tutorials/plot_compute_covariance.py

```diff
@@ -33,26 +34,40 @@
 # useful if you use resting state as a noise baseline. Here we use the whole
 # empty room recording to compute the noise covariance (tmax=None is the same
 # as the end of the recording, see :func:`mne.compute_raw_covariance`).
+#
+# Keep in mind that you want to match your empty room dataset to your
+# actual MEG data, processing-wise. Ensure that filters
+# are all the same and if you use ICA, apply it ot both your noise and
+# your MEG recording. In this case we did not filter the data and
```

Comment:
"to both your noise and your MEG recording" is a bit odd. Maybe "to your empty-room and subject data equivalently"?